### PR TITLE
glances: version bump to 3.1.5

### PR DESCRIPTION
**Pull request by kapsh**

### CODE REVIEW

File: packages/sys-process/glances/glances-3.1.5.exheres-0 [hunk 1]
```diff
@@ -7,5 +7,3 @@ require glances
 SLOT="0"
 PLATFORMS="~amd64"
 
-BUGS_TO="will@worrbase.com"
```
Comment (kapsh) [inline packages/sys-process/glances/glances-3.1.5.exheres-0:10]: Don't know the whole story, but BUGS_TO was removed from recommended metadata some time ago in https://gitlab.exherbo.org/exherbo-misc/docs/-/commit/8886e87da9dfd3818892e334f2c96d83d1d6fb7d

File: packages/sys-process/glances/glances.exlib [hunk 2]
```diff
@@ -17,52 +17,44 @@ MYOPTIONS=""
...
         ) [[
             *group-name = web
-            description = [ Web plugin ]
-        ]]
-        (
-            dev-python/matplotlib[python_abis:*(-)?]
-        ) [[
-            *group-name = chart
-            description = [ Chart plugin ]
+            *description = [ Web plugin ]
```
Comment (kapsh) [inline packages/sys-process/glances/glances.exlib:28]: Starred description makes nicer suggestions, as in
```
    Reasons: sys-process/glances-3.1.5:0::worr (sug): "Docker plugin"
    Take using: --take (docker)
```

File: packages/sys-process/glances/glances.exlib [hunk 3]
```diff
@@ -17,52 +17,44 @@ MYOPTIONS=""
...
     suggestion:
         (
             dev-python/bottle[python_abis:*(-)?]
             dev-python/requests[python_abis:*(-)?]
         ) [[
             *group-name = web
-            description = [ Web plugin ]
-        ]]
-        (
-            dev-python/matplotlib[python_abis:*(-)?]
```
Comment (kapsh) [inline packages/sys-process/glances/glances.exlib:30]: mpl is no more since https://github.com/nicolargo/glances/issues/1206

**Review by worr — APPROVED** [review]
To me, this looks good. Thanks for the doc references.

However, I'd recommend that you move this into your own exheres repo, as mine has lapsed substantially since I've moved all my systems back to OpenBSD.

**Comment by kapsh** [comment]
Appreciated. I've imported it into my repo (in unofficial) as https://gitlab.exherbo.org/kapsh/platypus/-/merge_requests/119 and posted another commit removing it from this.

**Pull request merged**